doc/lua_api.md

```diff
@@ -6043,6 +6044,14 @@ Item handling
 * `minetest.fourdir_to_dir(fourdir)`
     * Convert a 4dir back into a vector aimed directly out the "back" of a
       node.
+* `minetest.rotate_facedir(rotation, vector_name, facedir)`
+    * Returns the facedir value that results from rotating `facedir` around an axis.
+    * `face_name` is one of `"x+"`, `"x-"`, `"y+"`, `"y-"`, `"z+"` or `"z-"`;
+      rotation is performed around the axis perpendicular to it.
```

Comment:
There is no `face_name`. And, “axis perpendicular to it” is just confusing IMO as `x+` is an axis already.